src/components/article-page/article-page.jsx

```diff
@@ -1,36 +1,58 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getArticleByID, getCommentsByArticleID } from "../../api";
+import {
+  getArticleByID,
+  getCommentsByArticleID,
+  patchVoteByArticleID,
+} from "../../api";
 import CommentCard from "../comment-card/comment-card";
+import "./article-page.css";
 
 function ArticlePage() {
   const { article_id } = useParams();
   const [currArticle, setCurrArticle] = useState({});
   const [commentsArray, setCommentsArray] = useState([]);
+  const [votes, setVotes] = useState(0);
+
   const [loadingArticle, setLoadingArticle] = useState(true);
-  const [loadingComments, setLoadingComments] = useState([]);
+  const [loadingComments, setLoadingComments] = useState(true);
 
-  useEffect(() => {
-    async function fetchArticleByID() {
-      try {
-        setLoadingArticle(true);
-        const articleFromAPI = await getArticleByID(article_id);
-        setCurrArticle(articleFromAPI);
-        setLoadingArticle(false);
-      } catch (err) {
-        console.log(err);
-      }
+  const [voted, setVoted] = useState(false);
+
+  async function fetchArticleByID() {
+    try {
+      setLoadingArticle(true);
+      const articleFromAPI = await getArticleByID(article_id);
+      setCurrArticle(articleFromAPI);
+      setLoadingArticle(false);
+
+      setVotes(articleFromAPI.votes); //is it bad to put this here if its not really related to fetching the article
```

Comment:
This isn't bad and will need doing but I would investigate it as it currently doesn't look like it's doing what its supposed to?
Unless all the articles do have 0 votes?